pylint_django/tests/settings.py

```diff
@@ -0,0 +1,12 @@
+SECRET_KEY = 'fake-key'
+
+INSTALLED_APPS = [
+    'django.contrib.auth',
```

Comment:
These two apps and the two middleware are required for Django's authentication features (e.g. `auth.User`): https://docs.djangoproject.com/en/3.0/topics/auth/#installation .